Bottom: 88e5b1835918fdc8b0f0f8a0425ab0063c68e989
Top:    4b1d32b688ebca2c383e8dfb4951918b9cea8b89
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-11-09 22:51:28 +0000

Refresh of try-a-new-different-sized-vbo-

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 436b63e..25bbd9f 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -134,10 +134,6 @@ hidgl_init_triangle_array (triangle_buffer *buffer)
    */
   buffer->use_map = false;
 
-  /* If using VBOs (but not mapping), we only need to this once */
-//  if (buffer->use_vbo && !buffer->use_map)
-//    glBufferData (GL_ARRAY_BUFFER, BUFFER_SIZE, NULL, GL_STREAM_DRAW);
-
   buffer->triangle_array = NULL;
   hidgl_reset_triangle_array (buffer);
 }
